client/src/components/MatchPerformanceRow.js

```diff
@@ -119,56 +120,95 @@ export default function MacthPerformanceRow(performanceObject) {
       '--commander-image-url': `url(${getCommanderImageUrl()})`
     }
 
+    function getImageClass() {
+      if (endtime) {
+        return 'flex-column image-column washed';
+      } else {
+        return 'flex-column image-column';
+      }
+    }
+
+    const navigateToCommanderScryfall = async () => {
```

Comment:
probably want to avoid direct scryfall api calls if possible since we have no control of them timing out or ip banning users. in this case you can just do "https://scryfall.com/card/" + commanderId and it will make a link to the same page you are querying for here